### PR TITLE
Social sign-in blocked style updates

### PR DESCRIPTION
**Pull request by rebecca-thompson**
## What does this change?
This implements the final design for the 'user blocked from social sign-in' error message. This message shows when the user tries to sign-in with Google, Facebook or Apple, but they already have an account with a password and they haven't validated their email address. If they are blocked from signing in with social, it also removes the social buttons from the sign-in page.

## Images
Screenshot 2021-11-04 at 13 54 19
## Accessibility

- [ ] [Tested with screen reader](https://guardian.github.io/source/?path=/docs/docs-06-accessibility--page#screen-readers)
- [ ] [Navigable with keyboard](https://guardian.github.io/source/?path=/docs/docs-06-accessibility--page#keyboard-navigation)
- [ ] [Colour contrast passed](https://guardian.github.io/source/?path=/docs/docs-06-accessibility--page#colour-contrast)
- [ ] [The change doesn't use only colour to convey meaning](https://guardian.github.io/source/?path=/docs/docs-06-accessibility--page#use-of-colour)

### CODE REVIEW

**Comment by rebecca-thompson** [comment]
[Force Sign in for Social - dev](https://trello.com/c/0wKdfb2I/3533-force-sign-in-for-social-dev)

**Comment by rebecca-thompson** [comment]
[Force Sign in for Social - dev](https://trello.com/c/0wKdfb2I/3533-force-sign-in-for-social-dev)

**Comment by rebecca-thompson** [comment]
[Force Sign in for Social - dev](https://trello.com/c/0wKdfb2I/3533-force-sign-in-for-social-dev)

**Review by coldlink — APPROVED** [review]

File: cypress-ete.sh [hunk 1]
```diff
@@ -23,5 +23,5 @@ else
   yarn build
   yarn start &
   yarn wait-on:server
-  yarn cypress open --env $CI_ENV --config '{"testFiles":["ete/*.js"]}'
+  yarn cypress open --env $CI_ENV --config '{"testFiles":["ete/*.ts"]}'
```
Comment (ob6160) [inline cypress-ete.sh:26]: Mahesh is also changing in #1109, which version do we want to go in? https://github.com/guardian/gateway/pull/1109/files#diff-d2ddbe5aa010ad73e29564ba8183233bcec8b3b6982bdb6fff47c8a82f0c7143R26

**Review by ob6160 — APPROVED** [review]

File: cypress-ete.sh (hunk 1 above)
Comment (coldlink) [reply cypress-ete.sh:26]: the version I use also matches subdirectories, which we want, could add it in here if this PR goes in first?

```suggestion
  yarn cypress open --env $CI_ENV --config '{"testFiles":["ete/**/*.ts"]}'
```

Comment (rebecca-thompson) [reply cypress-ete.sh:26]: will do 👌

**Pull request merged**